src/allmydata/test/test_magic_folder.py

```diff
@@ -160,12 +160,12 @@ def _check_file(self, name_u, data, temporary=False, directory=False):
             # is renamed into place, which causes events that the test is not expecting.
             f = open(path_u, "wb")
             try:
-                if temporary and sys.platform != "win32":
+                if temporary and (sys.platform != "win32" or sys.platform != "darwin"):
                     os.unlink(path_u)
                 f.write(data)
             finally:
                 f.close()
-            if temporary and sys.platform == "win32":
+            if temporary and (sys.platform == "win32" or sys.platform == "darwin"):
```

Comment:
This is correct already, but make it ``and sys.platform in ("win32", "darwin")`` for consistency with above.